webapp/entrypoint.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 set -e
-export AZURE_HTTP_USER_AGENT="GITHUBACTIONS_${GITHUB_ACTION}"
+export AZURE_HTTP_USER_AGENT="GITHUBACTIONS_${GITHUB_ACTION}_${GITHUB_REPOSITORY}"
```

Comment:
Do you need to consume this var in kudu npm as well?

Reply:
good suggestion. 
Will take it in V2 version.

Reply:
sure. Make sense.